### ISSUE TITLE
review HIL

### ISSUE THREAD

**Issue by dagar** [PX4/Firmware#14449]
"when i run HITL simulations on latest master, i get sensor_baro:0 failed: TIMEOUT!, reconfiguring, messages. The attitude estimation (while landed) doesnt work, also the position and altitude.
I am usually runing HITL simulations on 1.10 release where it works fine."

Steps to reproduce
 - uploaded master to the fcu (hitl enabled, hil x airframe set)
 - compiled gazebo, changed iris model to use serial and enabled hitl
 - run gzserver

**Comment by dagar** [comment]
@dusan19

**Comment by LorenzMeier** [comment]
Can we get a "make abc" target for this?

**Comment by dagar** [comment]
> Can we get a "make abc" target for this?

Yes. We should also look for a way to set some of these things at runtime so we don't end up with more copied gazebo models for every single possible setup.

**Comment by LordHelmchen324** [comment]
I think this may be related to #14721.

After pulling the latest master and maybe doing some other updates (unfortunately I'm really not sure when it changed) I now, too, get the message `sensor_baro:0 failed: TIMEOUT!, reconfiguring priorities`.

**Comment by dagar** [comment]
Thanks for the update @LordHelmchen324. I'd like to get this fixed for the upcoming v1.11 release.
